Network/HTTP/Req.hs

```diff
@@ -155,6 +155,7 @@ module Network.HTTP.Req
     -- $query-parameters
     (=:),
     queryFlag,
+    formToQuery,
```

Comment:
I would not expose this as a standalone function.

Reply:
Sorry, what else can I do about it?

Reply:
Keep it internal to define `FromForm` but do not export it.

Reply:
I see. However, I think `formToQuery` would be significantly more beneficial for the user of this library than `FromForm` alone.

As described in #126, the main use case of `formToQuery` is to encode a user-defined record type as parameters. For example,

```haskell
data Person = Person { name :: String, age :: Int } deriving (Generic)

instance ToForm Person

postPerson :: (MonadHttp m) => Person -> m IgnoreResponse
postPerson p = req POST (https "example.com" /: "persons") (ReqBodyUrlEnc $ formToQuery p) ignoreResponse mempty
```

With `FromForm` instance alone, the body parameter would be

```haskell
(ReqBodyUrlEnc $ either (const $ error "this must not happen") id $ fromForm $ toForm p)
```

`formToQuery` would save a lot of boilerplate codes.